geopyspark/geotrellis/layer.py

```diff
@@ -33,9 +33,11 @@
                                    HashPartitionStrategy,
                                    SpatialPartitionStrategy,
                                    SpaceTimePartitionStrategy,
-                                   RasterizerOptions)
+                                   RasterizerOptions,
+                                   catalog)
```

Comment:
Instead of importing `catalog` could you try importing `write` instead? It's not really clear to me what's going on with those two jobs, so maybe that'll fix it?